modules/administration/pages/tuning-changelogs.adoc

```diff
@@ -12,26 +12,27 @@ The parameter defaults to [systemitem]``0``, which means unlimited.
 java.max_changelog_entries = 0
 ----
 
-If you set this parameter, it will come into effect only for new packages when they are synchronized.
-You might like to delete the cached data to remove older data.
+If you set this parameter (remember you need to restart the services with `spacewalk-service restart`), it will come into effect only for new packages when they are synchronized.
+
+You might like to delete and regenerate the cached data to remove older data.
```

Comment:
Instead of "delete and regenerate" I'd use the term "purge".  Then in the IMPORTANT note, we can explain that purging in this context means deleting and then regenerating.

Reply:
why?

Reply:
@Loquacity: Why... what?

If it's about my change: because the change comes into effect only for new packages. For existing packages, "delete and regenerate" is needed so the old changelogs are removed from database, and then regenerated with the limits specified at the config file.

Reply:
I meant why use "purge"?

Reply:
That's what the user wants to do, IMO.  Person does not want to delete and regenerate (that's the command sequence to be run), but purging is actually wanted.  Maybe, there is a better term.

Of course, it is perfectly fine if we explain how this purging takes place technically.

I do not insist on my proposal.  Maybe, it causes more trouble than help.